src/test/resources/features/actions/line_mover.feature

```diff
@@ -413,3 +464,292 @@ Feature: Line mover
+    """
+    <rule_like2> NAME2:
+        input: <content2>
+        output: <content2>
+
+    <comment>
+
+    <rule_like1> NAME1:
+        input: <content1>
+        output: <content1>
+    """
+    Examples:
+      | rule_like1  | rule_like2  | content1       | content2      | comment   |
+      | rule        | rule        | "file1.txt"    | "file2.txt"   | # comment |
+      | checkpoint  | rule        | "file1.txt"    | "file2.txt"   | # comment |
+      | rule        | checkpoint  | "file1.txt"    | "file2.txt"   | # comment |
+      | checkpoint  | checkpoint  | "file1.txt"    | "file2.txt"   | # comment |
+      | rule        | rule        | "file1.txt"    | \n"file2.txt" | # comment |
+      | checkpoint  | rule        | "file1.txt"    | \n"file2.txt" | # comment |
+      | rule        | checkpoint  | "file1.txt"    | \n"file2.txt" | # comment |
+      | checkpoint  | checkpoint  | "file1.txt"    | \n"file2.txt" | # comment |
+      | rule        | rule        | \n"file1.txt"  | "file2.txt"   | # comment |
+      | checkpoint  | rule        | \n"file1.txt"  | "file2.txt"   | # comment |
+      | rule        | checkpoint  | \n"file1.txt"  | "file2.txt"   | # comment |
+      | checkpoint  | checkpoint  | \n"file1.txt"  | "file2.txt"   | # comment |
+      | rule        | rule        | \n"file1.txt"  | \n"file2.txt" | # comment |
+      | checkpoint  | rule        | \n"file1.txt"  | \n"file2.txt" | # comment |
+      | rule        | checkpoint  | \n"file1.txt"  | \n"file2.txt" | # comment |
+      | checkpoint  | checkpoint  | \n"file1.txt"  | \n"file2.txt" | # comment |
+      | rule        | rule        | "file1.txt"    | "file2.txt"   |           |
+      | checkpoint  | rule        | "file1.txt"    | "file2.txt"   |           |
+      | rule        | checkpoint  | "file1.txt"    | "file2.txt"   |           |
+      | checkpoint  | checkpoint  | "file1.txt"    | "file2.txt"   |           |
+      | rule        | rule        | "file1.txt"    | \n"file2.txt" |           |
+      | checkpoint  | rule        | "file1.txt"    | \n"file2.txt" |           |
+      | rule        | checkpoint  | "file1.txt"    | \n"file2.txt" |           |
+      | checkpoint  | checkpoint  | "file1.txt"    | \n"file2.txt" |           |
+      | rule        | rule        | \n"file1.txt"  | "file2.txt"   |           |
+      | checkpoint  | rule        | \n"file1.txt"  | "file2.txt"   |           |
+      | rule        | checkpoint  | \n"file1.txt"  | "file2.txt"   |           |
+      | checkpoint  | checkpoint  | \n"file1.txt"  | "file2.txt"   |           |
+      | checkpoint  | rule        | \n"file1.txt"  | \n"file2.txt" |           |
+      | rule        | checkpoint  | \n"file1.txt"  | \n"file2.txt" |           |
+      | checkpoint  | checkpoint  | \n"file1.txt"  | \n"file2.txt" |           |
+      | rule        | rule        | \n"file1.txt"  | \n"file2.txt" |           |
+
+  Scenario Outline: Swap rule between rule sections
```

Comment:
Scenario outline with the same name already defined above

Reply:
I changed it